docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 
 
-# Example applications
+# DeepReg Demos
 `DeepReg` provides a collection of demonstrations, _DeepReg Demos_, using open-accesible data with real clinical applications.
 
 ## Paired images
-If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical application for tracking organ motion and other temporal changes between two time points require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expritory and inspritory, is such an example of both unsupervised (without labels) and combined supervision (with segmentation of anatomical structures). Furthermore, registering prostate MR, aquired before surgery, and the intra-operative trasnrectal ultrasound images is an example of weakly-supervised training for multimodal image registration.
+If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical application for tracking organ motion and other temporal changes between two time points require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expiratory and inspiratory, is such an example of both unsupervised (without labels) and combined supervision (trained wwith label dissimilarity based on segmentation of anatomical structures). Furthermore, registering prostate MR, aquired before surgery, and the intra-operative trasnrectal ultrasound images is an example of weakly-supervised training for multimodal image registration. Another DeepReg Demo of MR-to-ultrasound image registration is to track brain tumour locations during neurosurgery, in which combining unsupervised and weakly-supervised losses may be useful.
 
 ## Unpaired images
-Unpaired images are found in applications such as single-modality inter-subject registration, with two dataset loaders for labelled and unlabelled image data. Registering different MR images of brain from different subjects has played a fundamental role in population studies in neuroimaging. An example registering 3D ultrasound images from different surgical patients is also provided, especially to demonstrate the support for experiments such as cross-validation in DeepReg.
+Unpaired images are found in applications such as _single-modality_ _inter-subject_ registration, with two dataset loaders for labelled and unlabelled image data. Registering different MR images of brain from different subjects has played a fundamental role in population studies in neuroimaging. Two applications in registering unpaired inter-subject CT images, for lung and abdominal organs, represent the type of registration that may benefit from added label supervision. An example registering 3D ultrasound images from different surgical patients is included in DeepReg Demos, additionally, to demonstrate the support for experiments such as cross-validation in DeepReg.
```

Comment:
"Registering different MR images of brain from different subjects has played a fundamental role in population studies in neuroimaging". -> I would remove "in neuroimaging" because it is redundant with "MR images of brain"